setup.cfg

```diff
@@ -41,6 +30,7 @@ include_trailing_comma = True
 force_grid_wrap = 0
 use_parentheses = True
 line_length = 88
+src_paths=.
```

Comment:
if it works without this setting, we can maybbe skip it?

Reply:
Skipped 👍